### PR TITLE
feat(experimental): `Textfield` add dropdown support

### PR DESCRIPTION
**Pull request by waterplea**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

Closes #

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/experimental/components/textfield/textfield.template.html [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <ng-container *ngIf="directive?.change$ | async"></ng-container>
 <ng-content select="input"></ng-content>
-<ng-content select="select"></ng-content>
```
Comment (vladimirpotekhin) [inline projects/experimental/components/textfield/textfield.template.html:3]: Is it expected deletion? this was necessary for native select dropdown on mobile

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/experimental/components/textfield/textfield.template.html (hunk 1 above)
Comment (waterplea) [reply projects/experimental/components/textfield/textfield.template.html:3]: Yes, I would refactor the selects separately, I have a plan )

File: projects/demo/src/modules/experimental/textfield/textfield.component.ts [hunk 2]
```diff
@@ -18,6 +18,13 @@ export class ExampleTuiTextfieldComponent {
 
     readonly example1: TuiDocExample = {
         HTML: import('./examples/1/index.html?raw'),
-        LESS: import('./examples/1/index.less?raw'),
+    };
+
+    readonly example2: TuiDocExample = {
+        HTML: import('./examples/1/index.html?raw'),
```
Comment (nsbarsukov) [inline projects/demo/src/modules/experimental/textfield/textfield.component.ts:24]:
```suggestion
        HTML: import('./examples/2/index.html?raw'),
```

File: projects/demo/src/modules/experimental/textfield/textfield.component.ts [hunk 3]
```diff
@@ -18,6 +18,13 @@ export class ExampleTuiTextfieldComponent {
...
+    };
+
+    readonly example3: TuiDocExample = {
+        HTML: import('./examples/1/index.html?raw'),
```
Comment (nsbarsukov) [inline projects/demo/src/modules/experimental/textfield/textfield.component.ts:28]:
```suggestion
        HTML: import('./examples/3/index.html?raw'),
```

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**